modules/twinklespeedy.js

```diff
@@ -5,18 +5,18 @@
 
 
 /*
- ****************************************
- *** twinklespeedy.js: CSD module
- ****************************************
- * Mode of invocation:     Tab ("CSD")
- * Active on:              Non-special, existing pages
- *
- * NOTE FOR DEVELOPERS:
- *   If adding a new criterion, add it to the appropriate places at the top of
- *   twinkleconfig.js.  Also check out the default values of the CSD preferences
- *   in twinkle.js, and add your new criterion to those if you think it would be
- *   good.
- */
+	 ****************************************
```

Comment:
I see a lot of extra tabs added in your patch. Would you mind removing them so that the diff only touches the code needed to fix the issues mentioned in the pull request?

Reply:
I think it's good to go now.